include/revng/Pipeline/Runner.h

```diff
@@ -203,6 +203,8 @@ class Runner {
   /// as all all their transitive dependencies
   llvm::Error invalidate(const Target &Target);
   llvm::Error invalidate(const pipeline::TargetInStepSet &Invalidations);
+  // Remove all the files that can be re-created by the pipeline
```

Comment:
1. Leave a blank line between unrelated method declarations.
2. Use Doxygen. This is the second time I report this, fix all the instances.